Controller/MediaAdminController.php

```diff
@@ -77,10 +77,11 @@ public function listAction(Request $request = null)
         $datagrid->setValue('context', null, $context);
 
         // retrieve the main category for the tree view
-        $category = $this->container->get('sonata.classification.manager.category')->getRootCategory($context);
+        $rootCategory = $this->container->get('sonata.classification.manager.category')->getRootCategory($context); 
```

Comment:
Than it's probably this one: 
`--- a/test.php
+++ b/test.php
@@ -109,4 +109,4 @@ class MediaAdminController extends Controller
             'csrf_token' => $this->getCsrfToken('sonata.batch'),
         ));
     }
-}
\ No newline at end of file
+}
`